int16 roll
int16 pitch
int16 yaw
float32 depth